.gitlab-ci.yml

```diff
@@ -28,5 +28,14 @@ variables:
 
 include:
   - local: '/script/gitlabci/job_base.yml'
-  - local: '/script/gitlabci/job_cuda.yml'
+  - local: '/script/gitlabci/job_cuda9.2.yml'
+  - local: '/script/gitlabci/job_cuda10.0.yml'
+  - local: '/script/gitlabci/job_cuda10.1.yml'
+  - local: '/script/gitlabci/job_cuda10.2.yml'
+  - local: '/script/gitlabci/job_cuda11.0.yml'
+  - local: '/script/gitlabci/job_cuda11.1.yml'
+  - local: '/script/gitlabci/job_cuda11.2.yml'
+  - local: '/script/gitlabci/job_cuda11.3.yml'
+  - local: '/script/gitlabci/job_cuda11.4.yml'
+  - local: '/script/gitlabci/job_cuda11.5.yml'
```

Comment:
Why is this necessary? Is there any advantage?

Reply:
- The files get long and it is hard to keep an overview.
- It is much easier to compare to setups for different CUDA versions because I can render both side by side.

Reply:
I see. So there is no: multiple files can run in parallel etc.? I thought there might be an advantage like that.

Reply:
All jobs running in parallel. The problem is that the CI has only 4 slots for NVIDIA jobs. This is limiting parallelism.

Please do not ask me why we have only 4 slots else I start :sob: